settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -116,6 +116,33 @@ object TypelevelSettingsPlugin extends AutoPlugin {
         case _ => Seq.empty
       }
     },
+    scalacOptions ++= {
+      scalaVersion.value match {
+        case V(V(2, 12, _, _)) =>
```

Comment:
```suggestion
        case V(V(2, 12, _, _)) =>
```

This `V(V` thing could be fixed with a specialized `unapply` e.g. `V.Scala` like I have `V.Tag`.